cocotb/share/makefiles/Makefile.inc

```diff
@@ -29,6 +29,15 @@
 
 # Common makefile included by everything
 
+ifndef COCOTB_MAKEFILE_INC_INCLUDED # Protect against multiple includes
+COCOTB_MAKEFILE_INC_INCLUDED = 1
+
+# Make sure to use bash for the pipefail option used in many simulator Makefiles
+SHELL := bash
+
+.PHONY: all
+all: sim
```

Comment:
I worry this is going to irritate users who want to define their own `all` target

Reply:
Can move those 2 to simulator Makefiles or Makefile.sim (than need to be added by user)

Reply:
I think you should:

* move the `sim:` target to the top, so that `make` by itself runs the simulation
* Move the `all` target to `Makefile.sim`

Reply:
Hope it is right now. Seems that everything still works.